lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -531,6 +538,11 @@ public static function invalidDiscriminatorColumnType($className, $type)
         return new self("Discriminator column type on entity class '$className' is not allowed to be '$type'. 'string' or 'integer' type variables are suggested!");
     }
 
+    public static function invalidDiscriminatorValue($className)
+    {
+        return new self("Discriminator value on entity class '$className' should be a valud 'string'");
```

Comment:
Haha, yes, definitely :) Although, I don't think Doctrine will accept this anyways, so that's why I am not fixing the tests as well :) I Think this PR will be declined :)
But, hay, thanks :)